backend/src/main/java/ch/zhaw/pm4/compass/backend/exception/DayAlreadyExistsException.java

```diff
@@ -0,0 +1,9 @@
+package ch.zhaw.pm4.compass.backend.exception;
+
+import ch.zhaw.pm4.compass.backend.model.DaySheet;
+
+public class DayAlreadyExistsException extends RuntimeException {
```

Comment:
```suggestion
public class DaySheetAlreadyExistsException extends RuntimeException {
```

Bitte anderswo noch kontrollieren und korrigieren.